script.js

```diff
@@ -0,0 +1,162 @@
+const allSpaces = document.querySelectorAll('.spaces')
+const allX = document.querySelectorAll('.xImg')
+const allO = document.querySelectorAll('.oImg')
+const startButton = document.querySelector('.start')
+
+startButton.addEventListener('click', (event) =>{
+    startGame()
+})
+
+
+console.log(allX);
+//makes varaible with start button
+let turn = 0;
+
+
+
+
+function addListen(){
+    for (let i = 0; i < allSpaces.length; i++){
+        allSpaces[i].addEventListener('click', (event) =>{
+            if(allSpaces[i].classList[1] === "empty")
+            { 
+                if(turn % 2 === 0)
+                {
+                    allSpaces[i].classList.remove('empty')
+                    allX[i].classList.remove('hidden')
+                    console.log("x");
+                    let win = checkWin()
+                    document.getElementById('turn').innerText = "O turn"
+                    turn++
+                    console.log(turn);
+                    if(turn === 9 && win !== true){
+                        console.log("Draw")
+                    }
+                }
+                else{
+                    allSpaces[i].classList.remove('empty')
+                    allO[i].classList.remove('hidden')
+                    console.log('o')
+                    checkWin()
+                    document.getElementById('turn').innerText = "X turn"
+                    turn++
+                    console.log(turn);
+                    if(turn ===9){
+                        console.log("Draw")
+                    }
+                }    
+            }
+        })
+    }
```

Comment:
if(turn % 2 === 0) I think would be better assigned to a player turn boolean ex. playerOneTurn = true and inside your if statement you can alternate between playerOneTurn = true and playerOneTurn = false (meaning its player 2's turn). Makes it easier to read. You can then just update the boxes innerText to either X and O so you don't have to do as much DOM manipulation